src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -247,6 +247,18 @@ In these cases, you can opt out of client-side routing on a per-link basis using
 
 These links will be ignored by the router and a full page navigation will occur.
 
+## Replace entries in the browser history
+
+Normally, each navigation will push a new entry to the browser's history. This allows navigation between pages using the browser's back and forward buttons. 
+
+It is also possible to prevent a new entry from being written to the browser history by overriding the current entry. This is useful for pages that are not meant to be navigated back to, such as a login page. You do this by adding a `data-astro-history='replace'` attribute to an anchor like so:
+
+```html
```

Comment:
```suggestion
```astro
```

Reply:
Hi @dreyfus92! Thank your for your review! I copied the style of the former paragraph. What are the benefits of `astro` over `html` (only in the context of style for code blocks in mdx files 😄) Would it better to convert all to `astro` instead of `html`

Reply:
To maintain consistency, this section of the documentation utilizes the Astro extension to reference isolated portions of the HTML, as demonstrated in this example. Here, we have a straightforward HTML tag, and we also employ the Astro extension.
https://github.com/withastro/docs/blob/f36b5ef47425a90c8049633b59bdacb4b8f968bd/src/content/docs/en/guides/view-transitions.mdx#L89

Reply:
lol, and I found as a copy template the only example that was different. I will change both of them.